### DISCUSSION TITLE
Duda VUE

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#457] [Q&A]
Buenas tardes para la práctica, es literal hacer mi mimos de la pastelería (aunque claro se puede mejorar el diseño y demás) pero se va a dividir en componente como explica y como está el live.

¿Tendría funcionalidad? O eso aún no, oh cosas sencilla (?)

**Comment by Ingrzs** [comment]
Pues tengo entendido ,si no tenia funcionalidad tu pagina, ahora si tendrás que meterle para tener los componentes.

**Reply by AngelCruzO** [reply]
Los componentes no tendrian que ver con funcionalidad puesto que son maquetación HTML y CSS, en su caso seria la parte de vuex con los estados para dar "funcionalidad"

**Reply by Ingrzs** [reply]
exacto a eso me referia "simular la funcionalidad" no me estaba rifiirendo a que ya le pondria servidor y todo eso

**Reply by GloriaCoralCerecedo** [reply]
Oh vale entiendo, gracias!

**Comment by martinGM05** [comment]
Emm si, vaya hasta cierto punto debido a que no se tiene un servidor para almacenar las cosas sin embargo con vuex podriamos darle funcionalidad a cuestion de una misma sesión. (Sin guardar en BD)

**Reply by GloriaCoralCerecedo** [reply]
A lo que te refieres es que pueda hacer funcionalidad por ejemplo, que lo que el cliente ponga en el  formulario, se pueda pasar a la tabla de pedido, y descontar del inventario, aunque claro solo sería un valor y no se guardiaria.

**Reply by martinGM05** [reply]
Exacto, justo eso.
